.github/workflows/ada.yml

```diff
@@ -0,0 +1,33 @@
+name: Ada (GNAT)
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'
```

Comment:
Thanks @theavege I don't want to run the CI the fixed intervals. Only on push or PR is enougth.